<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Visuals</class>
 <widget class="QWidget" name="Visuals">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>121</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>121</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QGroupBox" name="orientation">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Orientation / supercell</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="translations" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>2</number>
         </property>
         <item row="0" column="1">
          <widget class="QLabel" name="xLabel">
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QToolButton" name="px">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="qagate.qrc">
             <normaloff>:/visuals/icons/plus-square.svg</normaloff>:/visuals/icons/plus-square.svg</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QToolButton" name="my">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="qagate.qrc">
             <normaloff>:/visuals/icons/minus-square.svg</normaloff>:/visuals/icons/minus-square.svg</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QToolButton" name="py">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="qagate.qrc">
             <normaloff>:/visuals/icons/plus-square.svg</normaloff>:/visuals/icons/plus-square.svg</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="yLabel">
           <property name="text">
            <string>y</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QToolButton" name="pz">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="qagate.qrc">
             <normaloff>:/visuals/icons/plus-square.svg</normaloff>:/visuals/icons/plus-square.svg</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QToolButton" name="mx">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="qagate.qrc">
             <normaloff>:/visuals/icons/minus-square.svg</normaloff>:/visuals/icons/minus-square.svg</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QToolButton" name="mz">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="qagate.qrc">
             <normaloff>:/visuals/icons/minus-square.svg</normaloff>:/visuals/icons/minus-square.svg</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="zLabel">
           <property name="text">
            <string>z</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="anglesTool" native="true">
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>2</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="thetaLabel">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="qagate.qrc">:/visuals/icons/theta.svg</pixmap>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="psiLabel">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="pixmap">
            <pixmap resource="qagate.qrc">:/visuals/icons/psi.svg</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="phiLabel">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="qagate.qrc">:/visuals/icons/phi.svg</pixmap>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QToolButton" name="z">
           <property name="text">
            <string>z</string>
           </property>
           <property name="iconSize">
            <size>
             <width>17</width>
             <height>17</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QToolButton" name="x">
           <property name="text">
            <string>x</string>
           </property>
           <property name="iconSize">
            <size>
             <width>17</width>
             <height>17</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QToolButton" name="y">
           <property name="text">
            <string>y</string>
           </property>
           <property name="iconSize">
            <size>
             <width>17</width>
             <height>17</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="phi">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="suffix">
            <string>°</string>
           </property>
           <property name="minimum">
            <number>-180</number>
           </property>
           <property name="maximum">
            <number>180</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="psi">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="suffix">
            <string>°</string>
           </property>
           <property name="minimum">
            <number>-180</number>
           </property>
           <property name="maximum">
            <number>180</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="theta">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="suffix">
            <string>°</string>
           </property>
           <property name="minimum">
            <number>-90</number>
           </property>
           <property name="maximum">
            <number>90</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="display">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Display</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="2">
       <widget class="QToolButton" name="axis">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qagate.qrc">
          <normaloff>:/visuals/icons/axis.svg</normaloff>:/visuals/icons/axis.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>17</width>
          <height>17</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="time">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qagate.qrc">
          <normaloff>:/visuals/icons/time.svg</normaloff>:/visuals/icons/time.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>17</width>
          <height>17</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QToolButton" name="angles">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Toggle angles</string>
        </property>
        <property name="statusTip">
         <string>Toggle angles</string>
        </property>
        <property name="whatsThis">
         <string>Toggle angles</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="qagate.qrc">
          <normaloff>:/visuals/icons/angles.svg</normaloff>:/visuals/icons/angles.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>17</width>
          <height>17</height>
         </size>
        </property>
        <property name="shortcut">
         <string>A</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>35</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="atom">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Atom / crystal</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="2" column="0">
       <widget class="QRadioButton" name="id">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Id</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="name">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="znucl">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Atomic num&amp;ber</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="bonds">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Bonds</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="atoms">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Atoms</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="none">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>None</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="borders">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Borders</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="physcis">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Physics</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <action name="actionAngles">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="qagate.qrc">
     <normaloff>:/visuals/icons/angles.svg</normaloff>:/visuals/icons/angles.svg</iconset>
   </property>
   <property name="text">
    <string>Angles</string>
   </property>
   <property name="toolTip">
    <string>Show/Hide angles</string>
   </property>
   <property name="statusTip">
    <string>Show/Hide angles</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="actionTime">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="qagate.qrc">
     <normaloff>:/visuals/icons/time.svg</normaloff>:/visuals/icons/time.svg</iconset>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
   <property name="toolTip">
    <string>Show/Hide time info</string>
   </property>
   <property name="statusTip">
    <string>Show/Hide time info</string>
   </property>
   <property name="whatsThis">
    <string>Show/Hide time info</string>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
  </action>
  <action name="action_px">
   <property name="icon">
    <iconset resource="qagate.qrc">
     <normaloff>:/visuals/icons/plus-square.svg</normaloff>:/visuals/icons/plus-square.svg</iconset>
   </property>
   <property name="text">
    <string>+x</string>
   </property>
   <property name="toolTip">
    <string>Add translation along x</string>
   </property>
   <property name="shortcut">
    <string>+, X</string>
   </property>
  </action>
  <action name="action_py">
   <property name="icon">
    <iconset resource="qagate.qrc">
     <normaloff>:/visuals/icons/plus-square.svg</normaloff>:/visuals/icons/plus-square.svg</iconset>
   </property>
   <property name="text">
    <string>+y</string>
   </property>
   <property name="toolTip">
    <string>Add translation along y</string>
   </property>
   <property name="shortcut">
    <string>+, Y</string>
   </property>
  </action>
  <action name="action_pz">
   <property name="icon">
    <iconset resource="qagate.qrc">
     <normaloff>:/visuals/icons/plus-square.svg</normaloff>:/visuals/icons/plus-square.svg</iconset>
   </property>
   <property name="text">
    <string>+z</string>
   </property>
   <property name="toolTip">
    <string>Add translation along z</string>
   </property>
   <property name="shortcut">
    <string>+, Z</string>
   </property>
  </action>
  <action name="action_mz">
   <property name="icon">
    <iconset resource="qagate.qrc">
     <normaloff>:/visuals/icons/minus-square.svg</normaloff>:/visuals/icons/minus-square.svg</iconset>
   </property>
   <property name="text">
    <string>-z</string>
   </property>
   <property name="toolTip">
    <string>Remove translation along z</string>
   </property>
   <property name="shortcut">
    <string>-, Z</string>
   </property>
  </action>
  <action name="action_my">
   <property name="icon">
    <iconset resource="qagate.qrc">
     <normaloff>:/visuals/icons/minus-square.svg</normaloff>:/visuals/icons/minus-square.svg</iconset>
   </property>
   <property name="text">
    <string>-y</string>
   </property>
   <property name="toolTip">
    <string>Remove translation along y</string>
   </property>
   <property name="shortcut">
    <string>-, Y</string>
   </property>
  </action>
  <action name="action_mx">
   <property name="icon">
    <iconset resource="qagate.qrc">
     <normaloff>:/visuals/icons/minus-square.svg</normaloff>:/visuals/icons/minus-square.svg</iconset>
   </property>
   <property name="text">
    <string>-x</string>
   </property>
   <property name="toolTip">
    <string>Remove translation along x</string>
   </property>
   <property name="shortcut">
    <string>-, X</string>
   </property>
  </action>
  <action name="actionX">
   <property name="text">
    <string>x</string>
   </property>
   <property name="toolTip">
    <string>Set view along x</string>
   </property>
   <property name="shortcut">
    <string>X</string>
   </property>
  </action>
  <action name="actionY">
   <property name="text">
    <string>y</string>
   </property>
   <property name="toolTip">
    <string>Set view along y</string>
   </property>
   <property name="shortcut">
    <string>Y</string>
   </property>
  </action>
  <action name="actionZ">
   <property name="text">
    <string>z</string>
   </property>
   <property name="toolTip">
    <string>Set view along z</string>
   </property>
   <property name="shortcut">
    <string>Z</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="qagate.qrc"/>
 </resources>
 <connections/>
</ui>
